addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -590,7 +618,10 @@ var SlideUploadDialog = Dialog.extend({
                     self.set('state', oldType);
                     self._alertDisplay(data.error);
                 } else {
-                    window.location = data.url;
+                    if (data.slide_type === "certification" && data.toast) {
+                        sessionStorage.setItem("toast", data.url);
+                    }
+                    window.location.reload();
```

Comment:
yes this is intended for all the slides once create the page is reloaded to see the new slide on the webpage in the list.
In the case of a certification, the specs ask that a toaster is shown with an url-link to the backend.
It could be a could idea to have the toaster for the other types of slides as well but I think this would be another task right?